game/plugin/skills/herblore/src/herblore.plugin.kts

```diff
@@ -0,0 +1,34 @@
+import org.apollo.game.message.impl.ItemOptionMessage
+import org.apollo.game.message.impl.ItemOnItemMessage
+
+on { ItemOptionMessage::class }
+        .where { option == 1 }
+        .then {
+            val unidentified = Herb[id] ?: return@then
+            it.startAction(IdentifyHerbAction(it, slot, unidentified))
+            terminate()
+        }
+
+on { ItemOnItemMessage::class }
+        .where { id == PESTLE_MORTAR || id == KNIFE }
+        .where { targetId == PESTLE_MORTAR || targetId == KNIFE }
```

Comment:
You can only apply where once, so you'll need to fold this into a single expression.

```
.where { (id == PESTLE_MORTAR || id == KNIFE) && (targetId == PESTLE_MORTAR || targetId == KNIFE) }
```

CC @Major- thoughts?

Reply:
We'll write some shim code to improve this as part of #395. Fine for now.